ee/api/feature_flag_role_access.py

```diff
@@ -0,0 +1,57 @@
+from rest_framework import exceptions, mixins, serializers, viewsets
+from rest_framework.permissions import SAFE_METHODS, BasePermission, IsAuthenticated
+
+from ee.models.feature_flag_role_access import FeatureFlagRoleAccess
+from ee.models.organization_resource_access import OrganizationResourceAccess
+from posthog.api.routing import StructuredViewSetMixin
+from posthog.models import FeatureFlag
+
+
+class FeatureFlagRoleAccessPermissions(BasePermission):
+    message = "You can't edit roles to this feature flag."
+
+    def has_permission(self, request, view):
+        if request.method in SAFE_METHODS:
+            return True
+        try:
+            resource_access = OrganizationResourceAccess.objects.get(
```

Comment:
same, filter on org here!

(unless I'm missing something and this happens automatically)

Reply:
I think this happens automatically but I'll double check!